PsychSourceGL/Cohorts/FTGLTextRenderer/OGLFT.cpp

```diff
@@ -3442,6 +3494,22 @@ QString Face::format_number ( const QString& format, double number ) { return( Q
     
     glEnd();
 
+    if (do_draw_underline_)
+    {
+        float undPos   = underline_position();
+        float undThicc = underline_thickness();
+	float advance = face->glyph->advance.x/64.f;
+	
+	glBindTexture( GL_TEXTURE_2D, 0);
+	glColor3f(0.f,0.f,0.f);
+        glBegin( GL_QUADS );
+        glVertex2f(0.f, undPos - undThicc);
```

Comment:
One may want to use glVertex2i instead of glVertex2f iff the rest of the quad positioning is done like that for the glyphs. Otherwise we might run into funny rounding-to-integral-pixel effects where the glyph does one thing, the underline the other, and you get weird wobble?

Reply:
ok

Reply:
Tested. If i switch to glVertex2i and corresponding static_cast<GLint>(something + .5f) for the arguments, i get wobble and uneven line thickness. All seems good (with my testing set) with glVertex2f. So keep as is i guess